modules/util.py

```diff
@@ -350,3 +377,26 @@ def mcopy(dev, src, dst):
         log.debug("Failed to mcopy in " + dev);
         return False
     return True
+
+def jsonAttributeExists(json, attribute):
```

Comment:
:) I will use the library. I thought you wanted to use bash commands as much as possible.